scipy/special/tests/test_cdflib.py

```diff
@@ -844,3 +843,38 @@ def test_chndtr_edge_cases(self, x, df, nc, expected):
 @pytest.mark.parametrize("x", [0.1, 100])
 def test_chdtriv_p_equals_1_returns_0(x):
     assert sp.chdtriv(1, x) == 0
+
+
+class TestPdtrik:
+    @pytest.mark.parametrize("p, m, expected",
+                             [(0, 0.5, 0),
+                              (0.5, 0, 0),
+                              (0, 0, 0)])
```

Comment:
The case p=0, m=0 is covered in `test_small_probabilities()` below.
```suggestion
                              (0.5, 0, 0)])
```